### ISSUE TITLE
Cannot change avatar placeholder text

### ISSUE THREAD

**Issue by l2aelba** [FaridSafi/react-native-gifted-chat#2225]
#### Issue Description

Can not change text style for placeholder in avatars

#### Steps to Reproduce / Code Snippets

```
import { Avatar } from 'react-native-gifted-chat'

function ChatAvatar(props) {
  return (
    <Avatar
      {...props}
      textStyle={{color: 'red '}}
    />
  )
}
```

#### Expected Results

Should able to style it

#### Additional Information

* Nodejs version: Latest
* React version: Latest
* React Native version: Latest
* react-native-gifted-chat version:  ^1.0.1
* Platform(s) (iOS, Android, or both?): both
* TypeScript version: n/a

**Comment by andrew-nguyen-mcs** [comment]
FYI: yeah, it looks like since this PR:
https://github.com/FaridSafi/react-native-gifted-chat/pull/1775

We've lost the ability to change the text style for placeholder avatars.  The textStyle prop isn't being passed down.

**Comment by KMathisGit** [comment]
I also am suffering from this loss of functionality.

**Comment by lyndseyjw** [comment]
yes, please update so that we may change `textStyle` on `Avatar`!!! specifically `font-family` 🥴
